src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -85,18 +85,7 @@ const appendViewLinks = (
     options: {
       label: loc('goback', 'login'),
       step: 'cancel',
-      // eslint-disable-next-line no-script-url
-      href: cancelStep?.action ? 'javascript:void(0)' : (baseUrl || '/'),
-      dataSe: 'cancel',
-      // @ts-ignore OKTA-512706 temporary until auth-js applies this fix
-      action: cancelStep?.action && ((params?: IdxActionParams) => {
-        const { stateHandle, ...rest } = params ?? {};
-        return widgetProps.authClient?.idx.proceed({
-          // @ts-ignore stateHandle can be undefined
-          stateHandle,
-          actions: [{ name: 'cancel', params: rest }],
-        });
-      }),
+      href: cancelStep ? undefined : (baseUrl || '/'),
```

Comment:
just confirming that we don't need to check `cancelStep.action` and only existence of `cancelStep` is enough here

Reply:
Step action fn cannot compatible with okta hosted scenario very well (due to missing stateToken), v3 will not depend on that function any more.
To your question, it won't be needed, as action is required in cancel step, so only check step will be enough.